packages/gatsby-theme-docs/gatsby-config.js

```diff
@@ -210,6 +213,54 @@ module.exports = (themeOptions = {}) => {
       'gatsby-plugin-remove-trailing-slashes',
       'gatsby-plugin-meta-redirect',
       'gatsby-plugin-netlify-cache',
+      {
+        resolve: 'gatsby-plugin-feed',
+        options: {
+          query: `
+          {
+            site {
+              siteMetadata {
+                title
+                description
+                siteUrl
+                site_url: siteUrl
+              }
+              pathPrefix
+            }
+          }
+        `,
+          feeds: [
+            {
+              serialize: ({ query: { site, allReleaseNotePage } }) => {
+                return allReleaseNotePage.nodes.map((node) => {
+                  return {
+                    ...node,
+                    url: `${site.siteMetadata.siteUrl}${site.pathPrefix}${node.slug}`,
+                    guid: `${site.siteMetadata.siteUrl}${site.pathPrefix}${node.slug}`,
+                  };
+                });
+              },
+              query: `
+              {
+                allReleaseNotePage(
+                  limit: 5, sort: { order: DESC, fields: date }, filter: {date: {gt: "1999-01-01"}},
+                ) {
+                    nodes {
+                      description
+                      slug
+                      title
+                      date
+                      categories: topics
+                    }
+                }
+              }
+            `,
+              output: '/release-notes.xml',
```

Comment:
@nkuehn just to clarify, will we have the following URL routes?

- `/releases`
- `/release-notes.xml`

Reply:
Yes.  If they are not horribly bad I keep the existing URL patterns.  But if you find them ugly and aren't scared of ending up with a big massive redirects setup for migration we can "clean it up", too and make it  `/releases.xml`. 
No real emotions, just push it.

Reply:
Ok, I was thinking more of something like `/releases/feed.xml`

Reply:
Fine too.  Including the XML extension prevents collisions and  `/feed`  was a conventional URL.